test/fixtures/linter/autofix/ansi/002_indentation/after.sql

```diff
@@ -8,14 +8,18 @@ SELECT
     (
         a.over_indented_line
     ) as bar,
-    a.line + (a.with
-              + a.hanging_indent) as actually_ok,
-    a.line + (a.with
-              + a.bad_hanging_indent) as problem,
+    a.line + (
+        a.with
+        + a.hanging_indent
+    ) as actually_ok,
+    a.line + (
+        a.with
+        + a.bad_hanging_indent
+    ) as problem,
     a.line + (
         a.something_indented_well
         + least(
-    	    a.good_example,
+            a.good_example,
```

Comment:
I wonder what changed here? Tab vs space, maybe?

Reply:
yeah - `reindent` also coerces indents to be of the right composition too.